packages/flame/lib/src/gestures/events.dart

```diff
@@ -0,0 +1,174 @@
+import 'package:flutter/gestures.dart';
+
+import '../../extensions.dart';
+import '../game/game.dart';
+
+class EventPosition {
+  final Game _game;
+  final Offset _localPosition;
```

Comment:
Can this be const?

Reply:
I am confused, this is a field that will be different across different instances, this couldn't be const. Or did I miss something here?